content/en/blog/2020/proxy-cert/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Provision a certificate and key to your application through files [Experimental]
+description: Provision a certificate and key to your application through files.
+publishdate: 2020-02-20
+attribution: Lei Tang (Google)
+keywords: [certificate,sidecar]
+target_release: 1.5
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio sidecars can obtain certificates through
+the secret discovery service.
+For an application without a sidecar to get a certificate,
+it may deploy a sidecar to provision the private key and certificates through
+the CSR flow from the CA and share the certificate with the application
+through a mounted file in `tmpfs`.
+This tutorial uses Prometheus as an example application to show provisioning
+a certificate through such mechanism.
```

Comment:
I think what is missing in the blog is why someone should do this vs a sidecar injector to inject the sidecar and have the certificates fetches by istio-agent from istiod.  What is the value to go through the complicated steps as described in this blog?

Reply:
Sorry this may be obvious to you but not for me.  Don't we have sidecar injector today to inject sidecar to services like productpage?  Why someone would need to use this manual approach vs that?

Reply:
cc @frankbu who may know more about the background

Reply:
The sidecar injector injects a sidecar for a service. The reason that someone would use this manual approach is to provision a certificate for the service (e.g., Prometheus) itself.